Automattic/vip-go-mu-plugins
# Jetpack 11.2 default

## Description
This change makes Jetpack 11.2 the default Jetpack version on the platform.

## Changelog Description
### Plugin Updated: Jetpack 11.2

We set Jetpack 11.2 as the default version of Jetpack on the VIP platform. All unpinned sites will now be using Jetpack 11.2

## Checklist


## Steps to Test

1. Check out PR.
1. Go to `wp-admin` > `Jetpack`
1. Verify the Jetpack version is 11.2.
